shop/src/test/java/ro/msg/learning/shop/TestCreateOrderConfig.java

```diff
@@ -0,0 +1,17 @@
+package ro.msg.learning.shop;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Profile;
+import ro.msg.learning.shop.service.OrderService;
+
+@TestConfiguration
+@Profile("test")
+public class TestCreateOrderConfig {
+
+//    @Bean
```

Comment:
👀  why?

Reply:
i tried somethig that didn t work and i forgot to delete , sorry :))